### ISSUE TITLE
Chat in a separate window

### ISSUE THREAD

**Issue by msouth** [ironbane/IronbaneServerLegacy#428]
- Chat is distracting
- Chat can be accidentally opened in the middle of a battle, killing the player
- Chat obscures the battle field when you need to aim for something in the upper right
- Working around this by closing the interface with 'u' also hides your vitals status (critical information in battle) and makes you unable to pick up items.
- It's lame to have the chat up while recording tutorials, etc
- I want to easily be able to look at the chat backlog after I'm done with a battle

Could you pop out a separate window, that does no 3d stuff or anything, just text with the chat?  Users could put it to the side of their browser window.

Could you have a bot that just reflects the chat to an ironbane-chat channel on freenode?  (Probably read-only in the first iteration--otherwise people could spam from there as well as in-game and it would be harder to deal with.)

**Comment by bsparks** [comment]
I was thinking of moving it to the bottom like this:
![maxresdefault 1](https://f.cloud.github.com/assets/384189/1851913/60d73f04-76f1-11e3-815f-01bcb2800e50.jpg)

making it much smaller and only showing the tabs and "full" details by clicking on something, so without interacting with it only show the last 2 or 3 lines.

possibly also making it a draggable / snappable type window, but that is more complex of a UI.

integrating with IRC would be nice, but I am not sure that freenode would like it if we used it for the in game chat. maybe we can setup our own irc server (irc.ironbane.com) to use for the chat (would be nice to use for the chat on the website too).

**Comment by ghost** [comment]
Agreed, chat is a real hassle, mostly due to its size and placement.

I think of of the main reasons that most MMOs place chat down below is that the text input can't be accidentally clicked as easily as it being nearby the non GUI portion of the screen.

Should also be able to be hidden and shown for sure.

Don't care at all for it being in another window, though. I guess if it's an option then players may use it occasionally, but it sounds like more hassle than it would be worth. Catering more to the minority of players. :P

**Comment by msouth** [comment]
Pop-out option like in gmail compose (or, maybe that was the old gmail compose)?
